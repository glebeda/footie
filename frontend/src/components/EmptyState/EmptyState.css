  .container {
    text-align: center;
  }
  
  @media only screen and (max-width: 600px) {
    .title {
      font-size: 1.5rem; 
    }
  }
  
  .imageContainer {
    display: flex;
    justify-content: center; 
    align-items: center; 
    min-height: 60vh; 
  }
  
  .image {
    max-width: 100%;
    height: auto;
  }
  