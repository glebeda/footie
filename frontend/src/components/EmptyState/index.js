import React from 'react';
import { Container, Typography, Box } from '@mui/material';
import disappointedSoccerPlayer from '../../assets/images/empty_state.png';
import './EmptyState.css';

const EmptyState = () => {
    return (
        <Container maxWidth='sm' className="container">
          <Typography variant='h4' component='h1' gutterBottom>
            Oi, no match is on the cards just yet
          </Typography>
          <Box className="imageContainer">
            <img src={disappointedSoccerPlayer} alt="Empty State" className="image" />
          </Box>
        </Container>
      );
};

export default EmptyState;