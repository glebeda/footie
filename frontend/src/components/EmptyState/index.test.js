import React from 'react';
import { render } from '@testing-library/react';
import EmptyState from './index';

describe('EmptyState', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<EmptyState />);
    expect(asFragment()).toMatchSnapshot();
  });
});
