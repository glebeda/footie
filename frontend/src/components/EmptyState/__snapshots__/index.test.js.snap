// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthSm page-container css-cuefkz-MuiContainer-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthSm container css-cuefkz-MuiContainer-root"
    >
      <h1
        class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom css-1vw6mcs-MuiTypography-root"
      >
        Oi, no match is on the cards just yet
      </h1>
      <div
        class="imageContainer MuiBox-root css-0"
      >
        <img
          alt="Empty State"
          class="image"
          src="test-file-name"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
